### ISSUE TITLE
Replace link to s5cmd with the link to the install instructions in the download manifest popup

### ISSUE THREAD

**Issue by fedorov** [ImagingDataCommons/IDC-WebApp#1284]
For those who are not developers, it may be helpful to have this link instead of the s5cmd github repo pointer, which we have right now: https://github.com/peak/s5cmd#installation.

**Comment by pgundluru** [comment]
Updated link in study and series export like below 
![image](https://github.com/ImagingDataCommons/IDC-WebApp/assets/85526819/f1ddf14a-65c3-410f-934a-d716b2dc0bfb)

**Comment by pgundluru** [comment]
@fedorov Sorry to open this back up, Did you also want the manifest themselves to have the link changed? For example in the below image the link still points to s5cmd Github?
![image](https://github.com/ImagingDataCommons/IDC-WebApp/assets/85526819/ee864c2c-a8a5-451a-b6ad-55f8a14b3e44)

**Comment by fedorov** [comment]
Yes, makes sense to point that to the installation instructions!

**Comment by s-paquette** [comment]
Since this is a backend change (files are constructed and then passed up to the front end) and we're close to release I'd like to bump this to release 41.

**Comment by fedorov** [comment]
This is related to #1399.

Now that the download instructions have been revised, the header should be the following:

```
# To download the files in this manifest, first install idc-index python 
# package (https://github.com/ImagingDataCommons/idc-index),
# then run the following command:
#
#   idc download manifest.s5cmd
#
# See IDC documentation for more details: https://learn.canceridc.dev/data/downloading-data
#
```
